docs/rst/tutorials/micro/user_api/user_api_parameter_server.rst

```diff
@@ -0,0 +1,341 @@
+
+        if (RCL_RET_OK != rc)
+        {
+            ...         // Handle error
+            return -1;
+        }
+
+- Integer:
+
+    .. code-block:: c
+
+        const char* parameter_name = "parameter_int";
+        int param_value = 100;
+
+        // Add parameter to the server
+        rcl_ret_t rc = rclc_add_parameter(&param_server, parameter_name, RCLC_PARAMETER_INT);
+
+        // Set parameter value
+        rc = rclc_parameter_set_int(&param_server, parameter_name, param_value);
+
+        // Get parameter value on param_value
+        rc = rclc_parameter_get_int(&param_server, parameter_name, &param_value);
+
+- Double:
+
+    .. code-block:: c
+
+        const char* parameter_name = "parameter_double";
+        double param_value = 0.15;
+
+        // Add parameter to the server
+        rcl_ret_t rc = rclc_add_parameter(&param_server, parameter_name, RCLC_PARAMETER_DOUBLE);
+
+        // Set parameter value
+        rc = rclc_parameter_set_double(&param_server, parameter_name, param_value);
+
+        // Get parameter value on param_value
+        rc = rclc_parameter_get_double(&param_server, parameter_name, &param_value);
+
+Max name size is controlled by the compile-time option `RCLC_PARAMETER_MAX_STRING_LENGTH`, default value is 50.
+
+Cleaning up
+^^^^^^^^^^^
+
+To destroy an initialized parameter server:
+
+.. code-block:: c
+
+    // Delete parameter server
+    rclc_parameter_server_fini(&param_server, &node);
+
+This will delete any automatically created infrastructure on the agent (if possible) and deallocate used memory on the parameter server side.
+
+micro-ROS Humble
+----------------
+
+Initialization options
+^^^^^^^^^^^^^^^^^^^^^^
+- Custom options:
+
+    The following options can be configured:
+
+    - notify_changed_over_dds: Publish parameter e  vents to other ROS 2 nodes as well.
+    - max_params: Maximum number of parameters allowed on the ``rclc_parameter_server_t`` object.
+    - allow_undeclared_parameters: Allows creation of parameters from external parameter clients. A new parameter will be created if a ``set`` operation is requested on a non-existing parameter.
+    - low_mem_mode: Reduces the memory used by the parameter server, functionality constrains are applied.
+
+    .. code-block:: c
+
+        // Parameter server object
+        rclc_parameter_server_t param_server;
+
+        // Initialize parameter server options
+        const rclc_parameter_options_t options = {
+            .notify_changed_over_dds = true,
+            .max_params = 4,
+            .allow_undeclared_parameters = true,
+            .low_mem_mode = false; };
+
+        // Initialize parameter server with configured options
+        rcl_ret_t rc = rclc_parameter_server_init_with_option(&param_server, &node, &options);
+
+        if (RCL_RET_OK != rc)
+        {
+            ...     // Handle error
+            return -1;
+        }
+
+- Low memory mode:
```

Comment:
Make this a subsection and explain that we have a low memory mode with the following functionality